test/inductor/test_torchinductor.py

```diff
@@ -342,6 +342,7 @@ def run(*ex, **kwargs):
         self.assertEqual(
             actual,
             correct,
+            msg="inductor: {}, correct: {}".format(actual, correct),
```

Comment:
I've intentionally added this printline. Please have a look at the output of this printline in https://github.com/pytorch/pytorch/actions/runs/3832510423/jobs/6523726036. Particularly, `inductor: tensor([1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568], dtype=torch.float16)` is obviously not random nor normal. 

```
Mismatched elements: 10 / 10 (100.0%)
Greatest absolute difference: 2.8798828125 at index (4,) (up to 1e-05 allowed)
Greatest relative difference: 12.64170616113744 at index (1,) (up to 0.001 allowed)
inductor: tensor([1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568, 1.7568,
        1.7568], dtype=torch.float16), correct: tensor([ 0.3367,  0.1288,  0.2345,  0.2303, -1.1230, -0.1863,  2.2090, -0.6382,
         0.4617,  0.2673], dtype=torch.float16)
------------------------------ Captured log call -------------------------------
WARNING  torch._inductor.lowering:lowering.py:1045 using triton random, expect difference from eager
------------------------------ Captured log call -------------------------------
WARNING  torch._inductor.lowering:lowering.py:1045 using triton random, expect difference from eager
```